src/core/bias-core.js

```diff
@@ -0,0 +1,51 @@
+import _ from 'lodash';
+import BPromise from 'bluebird';
+const {knex} = require('../util/database').connect();
+
+
+const calculateBias = (opts) => {
+  const wilsonsSql = `
+    SELECT
+      wilsons(
+        COUNT(CASE votes.value WHEN 1 THEN 1 ELSE null END)::numeric,
+        COUNT(CASE votes.value WHEN -1 THEN 1 ELSE null END)::numeric
+      )
+    FROM votes
+    LEFT JOIN feed_items ON feed_items.id = votes.feed_item_id
+    LEFT JOIN users ON users.id = feed_items.user_id
+    LEFT JOIN teams ON teams.id = users.team_id
```

Comment:
These all fields used in joins are NOT NULL, so these could be `JOIN`s instead of `LEFT JOIN`s